.github/workflows/tests.yml

```
@@ -63,6 +63,22 @@ jobs:
 
       - name: Run tests
         run: yarn test:integration
+  medusa-tests:
```

The tiniest nit: separate the actions with a blank line
```suggestion

  medusa-tests:
```